client/src/app/env.ts

```diff
@@ -0,0 +1,30 @@
+const ENV_VAR_KEYS = [
+  "AUTH_REQUIRED",
+  "KEYCLOAK_REALM",
+  "KEYCLOAK_CLIENT_ID",
+  "KEYCLOAK_SERVER_URL",
+  "UI_INGRESS_PROXY_BODY_SIZE",
+  "PROFILE",
+] as const;
+
+export type EnvVars = Record<typeof ENV_VAR_KEYS[number], string>;
+
+const ENV_DEFAULTS: EnvVars = {
+  AUTH_REQUIRED: "true",
+  KEYCLOAK_REALM: "",
+  KEYCLOAK_CLIENT_ID: "",
+  KEYCLOAK_SERVER_URL: "",
```

Comment:
We could also add those defaults : 
KEYCLOAK_REALM="tackle"
KEYCLOAK_CLIENT_ID="tackle-ui"